src/components/MyProfileActivityList/index.js

```diff
@@ -114,30 +125,33 @@ class MyProfileActivityList extends Component {
               lg={cardContentLengths.privacy.lg}
               xl={cardContentLengths.privacy.xl}
             >
-              <Grid container>
-                <Grid item xs={12} align="center">
-                  {/* The function you are trying to fire by clicking the toggle must passed to onChange()
+              {this.props.network === 'online' && (
+                <Grid container>
+                  <Grid item xs={12} align="center">
+                    {/* The function you are trying to fire by clicking the toggle must passed to onChange()
               using an Arrow Function.
               https://stackoverflow.com/questions/33846682/react-onclick-function-fires-on-render
               */}
-                  <Switch
-                    onChange={() => {
-                      this.handleChangeMembershipPrivacy(Membership);
-                    }}
-                    checked={!Membership.Privacy}
-                    classes={{
-                      switchBase: classes.colorSwitchBase,
-                      checked: classes.colorChecked,
-                      track: classes.colorBar,
-                    }}
-                  />
-                </Grid>
-                <Grid item xs={12} align="center">
-                  <Typography style={toggleTextStyle}>
-                    {Membership.Privacy ? 'Private' : 'Public'}
-                  </Typography>
+                    <Switch
+                      onChange={() => {
+                        this.handleChangeMembershipPrivacy(Membership);
+                      }}
+                      checked={!Membership.Privacy}
+                      classes={{
+                        switchBase: classes.colorSwitchBase,
+                        checked: classes.colorChecked,
+                        track: classes.colorBar,
+                      }}
+                    />
+                  </Grid>
+
+                  <Grid item xs={12} align="center">
+                    <Typography style={toggleTextStyle}>
+                      {Membership.Privacy ? 'Private' : 'Public'}
+                    </Typography>
+                  </Grid>
                 </Grid>
-              </Grid>
+              )}
```

Comment:
There's a check that happens to see if the user is offline. If so, the switch that would allow them to change the privacy of an activity is removed. Visually, it's actually hidden because the space that switches occupied remains visible while the switch itself is removed. This is to prevent the spacing of the card from changing when offline.